translations/harbour-foilnotes-pl.ts

```diff
@@ -41,7 +41,7 @@
     <message id="foilnotes-menu-show_qrcode">
         <source>Show QR code</source>
         <extracomment>Show QR code for the current note</extracomment>
-        <translation type="unfinished">Pokaż kod QR</translation>
+        <translation>Pokaż kod QR dla bieżącej notatki</translation>
```

Comment:
It's "Show QR code" that needs to be translated, "Show QR code for the current note" is just a comment, it's too long for a pulley menu item.

Reply:
Sorry about that. It's fixed now.